app/models/conversation.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Conversation < ApplicationRecord
+  acts_as_paranoid
```

Comment:
You should add the delete_at field to the messages table for soft deletion.
[Link to documentation](https://github.com/ActsAsParanoid/acts_as_paranoid#create-migration)